packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/iotSiteWiseQueryEditor.tsx

```diff
@@ -1,33 +1,83 @@
-import { type IoTSiteWiseClient } from '@aws-sdk/client-iotsitewise';
-import { type IoTTwinMakerClient } from '@aws-sdk/client-iottwinmaker';
+import {
+  IoTSiteWise,
+  IoTSiteWiseClient,
+  PropertyDataType,
+} from '@aws-sdk/client-iotsitewise';
 import Tabs from '@cloudscape-design/components/tabs';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { useSelector } from 'react-redux';
-
 import type { DashboardState } from '~/store/state';
-import { ModeledDataStreamQueryEditor } from './modeledDataStreamQueryEditor';
-import { UnmodeledDataStreamQueryEditor } from './unmodeledDataStreamExplorer';
 import { QueryExtender } from './queryExtender';
 import type { ModeledDataStream } from './modeledDataStreamQueryEditor/modeledDataStreamExplorer/types';
 import type { UnmodeledDataStream } from './unmodeledDataStreamExplorer/types';
 import { useQuery } from '../useQuery';
 import { AssetModelDataStreamExplorer } from './assetModelDataStreamExplorer/assetModelDataStreamExplorer';
+import {
+  AssetExplorer,
+  AssetExplorerProps,
+  AssetPropertyExplorer,
+  AssetPropertyExplorerProps,
+  TimeSeriesExplorer,
+  TimeSeriesExplorerProps,
+} from '@iot-app-kit/react-components';
+import { IoTTwinMakerClient } from '@aws-sdk/client-iottwinmaker';
+import { ResourceExplorerFooter } from './footer/footer';
+import { getPlugin } from '@iot-app-kit/core/dist/es/plugins/pluginsRegistry';
+import { Box } from '@cloudscape-design/components';
```

Comment:
Make sure you're importing components from Cloudscape like:

```ts
import Box from '@cloudscape-design/components/box';
```

The way you are importing is not tree-shakeable.